ui/window.go

```diff
@@ -865,6 +877,10 @@ func NewWindow(app *tview.Application, session *discordgo.Session, readyEvent *d
 		bottomBar := components.NewBottomBar()
 		bottomBar.AddItem(fmt.Sprintf("Logged in as: '%s'", tviewutil.Escape(session.State.User.Username)))
 		bottomBar.AddItem(fmt.Sprintf("View / Change shortcuts: %s", shortcutdialog.EventToString(shortcutsDialogShortcut)))
+		// bottomBar.AddItem(fmt.Sprintf("Vim: %s", window.app.VimMode.EnabledString()))
```

Comment:
Please don't commit code that's commented out. Review your own code before commiting, that's good practice.